fdbserver/workloads/FastTriggeredWatches.actor.cpp

```diff
@@ -145,7 +145,7 @@ struct FastTriggeredWatchesWorkload : TestWorkload {
 				//TraceEvent("FTWWatchDone").detail("Key", printable(setKey));
 				// Assert that the time from setting the key to triggering the watch is no greater than 25s
 				// TODO: This assertion can cause flaky behaviour since sometimes a watch can take longer to fire
```

Comment:
I'm not sure if this TODO is actually accurate. It's quite possible all of the cases we saw of the watch firing slowly were actually not timing the watch at all. Perhaps we could delete it and revisit if it recurs.

Reply:
Removed